src/pkg/cli/cert.go

```diff
@@ -211,41 +211,50 @@ func waitForCNAME(ctx context.Context, domain string, targets []string, client c
 		defer cancelSpinner()
 	}
 
+	verifyDNS := func() error {
+		if !serverSideVerified && serverVerifyRpcFailure < 3 {
+			if err := client.VerifyDNSSetup(ctx, &defangv1.VerifyDNSSetupRequest{Domain: domain, Targets: targets}); err == nil {
+				term.Debugf("Server side DNS verification for %v successful", domain)
+				serverSideVerified = true
+			} else {
+				if cerr := new(connect.Error); errors.As(err, &cerr) && cerr.Code() == connect.CodeFailedPrecondition {
+					term.Debugf("Server side DNS verification negative result: %v", cerr.Message())
+				} else {
+					term.Debugf("Server side DNS verification request for %v failed: %v", domain, err)
+					serverVerifyRpcFailure++
+				}
+			}
+			if serverVerifyRpcFailure >= 3 {
+				term.Warnf("Server side DNS verification for %v failed multiple times, skipping server side DNS verification.", domain)
+			}
+		}
+		if serverSideVerified || serverVerifyRpcFailure >= 3 {
+			locallyVerified := dns.CheckDomainDNSReady(ctx, domain, targets)
+			if serverSideVerified && !locallyVerified {
+				term.Warnf("The DNS configuration for %v has been successfully verified. However, your local environment may still be using cached data, so it could take several minutes for the DNS changes to propagate on your system.", domain)
+				return nil
+			}
+			if locallyVerified {
+				return nil
+			}
+		}
+		return errors.New("not verified")
+	}
+
+	if err := verifyDNS(); err == nil {
+		return nil
+	}
+	term.Infof("Please set up a CNAME record for %v", domain)
+	fmt.Printf("  %v  CNAME or as an alias to [ %v ]\n", domain, strings.Join(targets, " or "))
+	term.Infof("Waiting for CNAME record setup and DNS propagation...")
```

Comment:
```suggestion
	term.Infof("Configure a CNAME or ALIAS record for the domain name: %v", domain)
	fmt.Printf("  %v  -> %v\n", domain, strings.Join(targets, " or "))
	term.Infof("Awaiting DNS record setup and propagation... This may take a while.")
```